examples/Server_With_All_Interfaces-Settings.js

```diff
@@ -19,6 +19,7 @@ var moscaSetting = {
         { type: "https", port: 3001, bundle: true, credentials: { keyPath: SECURE_KEY, certPath: SECURE_CERT } }
     ],
     stats: false,
+    qos2Puback: false, // can set to true if using a client which will eat puback for QOS 2; e.g. mqtt.js
```

Comment:
I would not worry about it, it should not be a hot path: QoS 2 is not really supported.